lib/ansible/modules/windows/win_acl.ps1

```diff
@@ -63,7 +63,7 @@ Function UserSearch
 
     if (($searchDomain -eq $false) -and ($SearchAppPools -eq $false))
     {
-        # do not use Win32_UserAccount, because e.g. SYSTEM (BUILTIN\SYSTEM or COMPUUTERNAME\SYSTEM) will not be listed. on Win32_Account groups will be listed too
+        # do not use Win32_UserAccount, because e.g. SYSTEM (BUILTIN\SYSTEM or COMPUTERNAME\SYSTEM) will not be listed. on Win32_Account groups will be listed too
         $localaccount = get-wmiobject -class "Win32_Account" -namespace "root\CIMV2" -filter "(LocalAccount = True)" | where {$_.Caption -eq $accountName}
```

Comment:
Why we need to do it with WMI, is the .Net Object not ok to use ?

System.Security.Principal.NTAccount($accountName) should returns Domain and local Accounts, and a SearchQuery for a NonTrusted Domain realy should not happening here.